SECP-service/src/main/resources/assets/app/views/login/authenticate.html

```diff
@@ -0,0 +1,17 @@
+<div class="container-fluid">
+    <div class="form-group">
+        <form ng-submit="authenticate()" name="authForm">
+            <h1 id="formHeader">Authenticate</h1>
```

Comment:
let's stay consistent throughout the app. This header should be on the left. Just like how Moe did for other views.